api/processors/auxil/log_event_creator.py

```diff
@@ -52,3 +65,87 @@ def create(  # noqa: PLR0913
             )
             for teacher in group.teachers.iterator()
         )
+
+
+class CoordinatorAdminLogEventCreator:
+    @staticmethod
+    def _get_working_project_status(coordinator: Coordinator) -> CoordinatorProjectStatus:
+        group_count = coordinator.groups.count()
+        if group_count < CoordinatorGroupLimit.MIN:
+            return CoordinatorProjectStatus.WORKING_BELOW_THRESHOLD
+        if CoordinatorGroupLimit.MAX > group_count >= CoordinatorGroupLimit.MIN:
+            return CoordinatorProjectStatus.WORKING_OK
+        if group_count >= CoordinatorGroupLimit.MAX:
+            return CoordinatorProjectStatus.WORKING_LIMIT_REACHED
+
+        raise ValueError(f"Unexpected group count: {group_count}, coordinator: {coordinator}")
```

Comment:
Maybe `NotImplementedError` and include limits in the error message?

Reply:
Done